scripts/lib/sphinx/sphinxHtmlToMarkdown.ts

```diff
@@ -64,6 +66,10 @@ export async function sphinxHtmlToMarkdown(options: {
     }
   });
 
+  if (isReleaseNotes && releaseNotesTitle) {
+    $page("h1").html(releaseNotesTitle);
+  }
```

Comment:
What does this do?

Reply:
This replaces the title "Release Notes" with something like "Qiskit 0.45 release notes", I'll add a comment.